app/workers/first_worker.rb

```diff
@@ -2,8 +2,12 @@
 
 class FirstWorker
   include Sidekiq::Worker
-  def perform(*args)
-    puts 'I am backgroud worker i tried to send and email new'
-    TodoMailer.weekly_report(todo)
+  def perform(*) #antes *
+  #  @user = User.all
+  #  @user.each do |u|
+  #  mail.deliver_now
+  #  mail = TodoMailer.weekly_report(u)
+  #  end
+    puts "hola first worker"
```

Comment:
please avoid adding unnecessary puts to your commits

Reply:
ok